docs/releases/1.10.txt

```diff
@@ -163,7 +163,11 @@ Migrations
 Models
 ^^^^^^
 
-* ...
+* Reverse foreign keys from proxy models are now propagated to their
+  concrete class. Before this change the reverse relation attached by a
```

Comment:
The reverse relation .... is now accessible... and may be referenced in queryset filtering.